.github/workflows/test.yaml

```diff
@@ -0,0 +1,38 @@
+name: build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+env:
+  IP: 127.0.0.1
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-version: [1.14.x]
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@master
+        with:
+          fetch-depth: 1
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go-version }}
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v2
+        with:
+          version: v1.29
+      - name: test faas-swarm
```

Comment:
Is it possible to split the setup, test, and tear down for swarm and faas-netes? If there are problems in the future it will be a little more clear why it  failed.

As an example, the faas-netes worflow has individual steps https://github.com/openfaas/faas-netes/blob/master/.github/workflows/build.yaml

Reply:
thank you @LucasRoesler , done.